samples/Microsoft.AspNetCore.Proxy.Samples/Microsoft.AspNetCore.Proxy.Samples.csproj

```diff
@@ -1,7 +1,7 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
```

Comment:
So currently AppVeyor fails with a Missing Method Exception. 
You can add 
```xml
<PropertyGroup>
<!-- TODO remove when https://github.com/Microsoft/vstest/issues/428 is resolved -->
  <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
  <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
</PropertyGroup>`
```
That's our current work around and that should fix it.